docs/source/noise/index.rst

```diff
@@ -177,10 +177,11 @@ Noise types for each column
     - Decennial Census, ACS, CPS, WIC
     - Leave a field blank, choose the wrong option
     -
-  * - SSN
-    - W-2 and 1099, SSA
+  * - SSN or ITIN
+    - W-2 and 1099, 1040, SSA
     - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
-    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset
+    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset; ITIN only included if no SSN present.
```

Comment:
```suggestion
    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.
      ITIN is only reported in the 1040, and only when the simulant does not have an SSN.
```

Maybe? I want to make it clear that there won't be ITIN in any other data source.